src/main/java/teammates/ui/template/FeedbackResponseCommentRow.java

```diff
@@ -259,8 +260,9 @@ private String getEditedAtText(String lastEditorEmail, Instant createdAt, Instan
         }
         boolean isGiverAnonymous = Const.DISPLAYED_NAME_FOR_ANONYMOUS_PARTICIPANT.equals(commentGiverName);
         return "(last edited "
-                + (isGiverAnonymous ? "" : "by " + instructorEmailNameTable.get(lastEditorEmail) + " ")
-                + "at " + TimeHelper.formatDateTimeForSessions(lastEditedAt, sessionTimeZone)
-                + ")";
+                + (isGiverAnonymous
+                    ? ""
+                    : "by " + SanitizationHelper.sanitizeForHtml(instructorEmailNameTable.get(lastEditorEmail)) + " ")
```

Comment:
Just a note. This is not relevant for this PR. However, it sanitises the **name** of the instructor in the displayed HTML page. Let's just leave it there.